### PR TITLE
Graphviz manual migration

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@twrightsman - this is ready for review/merge :)

File: recipe/build.sh [hunk 1]
```diff
@@ -4,7 +4,8 @@ set -ex
 cmake . \
   -DUSE_PYTHON=ON \
   -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
-  -DCMAKE_INSTALL_LIBDIR=lib
+  -DCMAKE_INSTALL_LIBDIR=lib \
+  -DPython_EXECUTABLE="$PYTHON"
```
Comment (hmaarrfk) [inline recipe/build.sh:8]: Can we try to use a hint instead of an output: https://cmake.org/cmake/help/latest/module/FindPython.html#hints

Comment (Tobias-Fischer) [reply recipe/build.sh:8]: Let's just use #15?

Comment (Tobias-Fischer) [reply recipe/build.sh:8]: By the way, the knowledge base suggests "my" way: https://conda-forge.org/docs/maintainer/knowledge_base.html

Comment (hmaarrfk) [reply recipe/build.sh:8]: I figured I was going to have some kind of typo.

**Pull request closed without merging**